### ISSUE TITLE
Remove unreferenced blobs/clean up repository

### ISSUE THREAD

**Issue by mitkin** [pytroll/mpop#25]
Currently the size of unpacked repository is about 28Mb. A big contributor to the size is the `etc/world_map.ascii` file, which is removed long time ago. By removing the blobs related to it the size of the (unpacked) repository drops to 8Mb.

Unfortunately removing the blob and all references using git filter-branch:

    git filter-branch --index-filter 'git rm --cached --ignore-unmatch etc/world_map.ascii' HEAD

rewrites history and changes commit SHAs, therefore it might be necessary for the developers to start with a clean clone in order to avoid conflicts and pushing dirty history back.

Consider doing cleaning exercise before releasing mpop v2.0.

**Comment by mitkin** [comment]
Won't fix until mpop is ready for the v2.0 release. Clean up should be done before tagging/releasing.

**Comment by mraspaud** [comment]
Done in the satpy repo
